resources/sass/overrides/_toolbar.scss

```diff
@@ -16,6 +24,13 @@
   --bs-offcanvas-zindex: 5070;
 }
 
+body:has(#globan) {
+  .bcl-inpage-navigation,
+  .bcl-sidebar {
+    top: 28px;
+  }
+}
+
```

Comment:
this is not needed in the end

Reply:
i talked with fra, globan can be sticky and we should keep.